src/omega/SConstruct

```diff
@@ -1,6 +1,14 @@
-Program(
+import os
+
+conan = SConscript('build/SConscript_conan')
+# To install it via Conan you need to add Bintry to your remotes:
+# conan remote add public-conan https://api.bintray.com/conan/bincrafters/public-conan 
+if "sfml" in conan:
+    conan['conan']['LIBS'].extend(['sfml-graphics', 'sfml-window', 'sfml-system'])
```

Comment:
This is incorrect - you don't need to hack things here if everything is installed fine via Conan in the first place.
TODO: fix this